test/remote/gateways/remote_priority_test.rb

```diff
@@ -0,0 +1,428 @@
+require 'test_helper'
+
+class RemotePriorityTest < Test::Unit::TestCase
+  def setup
+    # Consumer API Key: Generated in MX Merchant for specific test merchant
+    # Consumer API Secret:= Generated in MX Merchant for specific test merchant
+
+    # run command below to run tests in debug (byebug)
+    # byebug -Itest test/unit/gateways/card_stream_test.rb
+    #
+    # bundle exec rake test:remote TEST=test/remote/gateways/remote_priority_test.rb
+    # ruby -Itest test/unit/gateways/priority_test.rb -n test_successful_void
+
+    # Run specific remote test
+    # ruby -Itest test/remote/gateways/remote_priority_test.rb -n test_fail_refund_already_refunded_purchase_response
```

Comment:
Thanks for these comments!! They're super helpful so I don't have to look it up 👏🏻 👏🏻